pkg/generators/openapi.go

```diff
@@ -376,8 +442,9 @@ func (g openAPITypeWriter) generate(t *types.Type) error {
 				"SchemaProps: spec.SchemaProps{\n", args)
 			g.generateDescription(t.CommentLines)
 			g.Do("Type:$.type|raw${}.OpenAPISchemaType(),\n"+
-				"Format:$.type|raw${}.OpenAPISchemaFormat(),\n"+
-				"},\n"+
+				"Format:$.type|raw${}.OpenAPISchemaFormat(),\n", args)
+			g.generateValueValidations(&overrides.SchemaProps)
```

Comment:
I see the following new calls to `generateValueValidations`

- [x] Custom V2 Type/Format and V3 Definition Functions
- [x] Custom V2 Type/Format and V3 oneOf types
- [ ] Custom V2 Type/Format (and No V3 Definition or OneOf)
- [ ] V3 OneOf Types
- [x] Regular Behavior (no custom types or override functions implemented)

The checked boxes are those that an integration exists already. This is function is pretty complicated. I wonder if we can simplify this function in a future PR. In the meantime, I'd like to see the rest of the cases covered in the integration tests to make sure the output compiles in those cases.

Reply:
resolved